### DISCUSSION TITLE
mollie/polyfill-libsodium is now abandoned

### DISCUSSION THREAD

**Discussion by Jamesking56** [discord-php/DiscordPHP#1247] [Q&A] [answered]
`mollie/polyfill-libsodium` is now abandoned and should be removed as a dependency from this package:

https://packagist.org/packages/mollie/polyfill-libsodium

**Comment by valzargaming** [comment]
This should be posted as an issue rather than discussion. I don't think it's used anywhere except for the voice client, but I also do not know if anyone is actively working on it anymore.

**Reply by Jamesking56** [reply]
Their issue templates suggest they only accept Bug Reports or Feature Requests. Anything else is forwarded to discussions

**Reply by valzargaming** [reply, marked as answer]
Discussions is intended for use specifically for non-library issues, such as how to use it rather than reporting an issue. Even so, this is probably a good forum to discuss this specific issue as it is a core dependency for the voice client. @key2peace and @CommandString are the only two people who I know that have used that side of the library at all, so my ability to weigh in on the topic is very limited.
